recipes/cubed/meta.yaml

```diff
@@ -2,17 +2,17 @@
 {% set version = "0.6.3" %}
 
 package:
-  name: "{{ name|lower }}"
-  version: "{{ version }}"
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 4f6716df51674634156224e249bdae0de7e8e0cd999a13c0faf8b00c68251e46
 
 build:
   number: 0
   noarch: python
-  script: "{{ PYTHON }} -m pip install . -vv"
+  script:"{{ PYTHON }} -m pip install . -vv
```

Comment:
oops. sorry!

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

Reply:
No problem! (This is my first time doing this myself so I appreciate the help!)